.gitignore

```diff
@@ -3,7 +3,7 @@ _build
 **/CMakeFiles
 **/CMakeCache.txt
 *.pyc
-sdk/
```

Comment:
Is this correct? The sdk directory is generated by scripts in hex/sd_api_vX.

Reply:
No, this is not correct, sdk should still be ignored.